Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+def erecipients = "TO_BE_FILLED"
+def ebody = """
+${currentBuild.fullDisplayName} / ${currentBuild.number}
+Check url: ${currentBuild.absoluteUrl}.
+"""
+
+def supported_8_machine_names = ["centos8", "generic-rhel8", "oracle8", "rockylinux8"]
```

Comment:
Since the upgrade script now support CentOS Stream support with new param `-d` option,  adding it as supported would be nice

Reply:
I have added CentOS Stream 8 but as separate array. The reasons are that firstly it's easier to implement that machine using a different command and it's future-proof - just add CentOS Stream 9 there once the migration script supports is.